City-of-Pori/pori-kada
# PORI-311 Fixed issue with Visitpori some teasers

To test:

1. drush cc all
2. Go to https://local.visitpori.fi/ and check that the facebook posts on front page look nice (no date under the logo or something like that)
